db/migrations/01_create_chitter_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, peep VARCHAR(280), peep_time timestamptz DEFAULT NOW());
```

Comment:
If you were to add user_id as a foreign key to this db, then you would be able to pull through the user against your peep.